dash/testing/browser.py

```diff
@@ -191,6 +194,19 @@ def wait_for_page(self, url=None, timeout=10):
                 )
             )
 
+    def toggle_window(self):
+        """switch between the current working window and the new opened one"""
+        if len(self.driver.window_handles) < 2:
+            raise BrowserError("there is no new opening window")
```

Comment:
```suggestion
            raise BrowserError("there is no second window")
```

Reply:
perhaps actually move this check into `switch_window`, and check for `len(...) <= idx`?